homeassistant/components/netgear/sensor.py

```diff
@@ -301,7 +301,7 @@ def __init__(
     ) -> None:
         """Initialize a Netgear device."""
         super().__init__(coordinator, router)
-        self.entity_description = entity_description
+        self.entity_description: NetgearSensorEntityDescription = entity_description
```

Comment:
Please move the type annotation for this attribute to a class attribute above the init method.

Reply:
done